ligo/src/types.mligo

```diff
@@ -129,10 +158,16 @@ type storage =
   ; proposal_key_list_sort_by_date : (timestamp * proposal_key) set
   ; permits_counter : nonce
   ; total_supply : total_supply
+  ; started_on : timestamp
+  ; freeze_history : freeze_history
+  ; vp_history : vp_history option
```

Comment:
As said above I suggest changing this to:
```suggestion
  ; freeze_history : freeze_history
  ; vp_history : vp_history
```

Also, I'd suggest to change the name from `vp_history` to `last_period_change` both for the type and field name.

Reply:
Done.